COMMERCE-10243 - PATCH in updating Specifications must have 'key' specified to succeed

victorcdp opened the pull request:
Related Issue: [COMMERCE-10243](https://issues.liferay.com/browse/COMMERCE-10243)

Problem was that there was no default value for the Specification Key when Patching.
Proposed fix is to use GetterUtil to get the Specification Key, so that if it is null, use the default value.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

matthewchan1 commented:
ci:test:sf

matthewchan1 commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-4-1.liferay.com/job/test-portal-source-format/1329/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-4-1.liferay.com//userContent/jobs/test-portal-source-format/builds/1329/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-22.liferay.com/job/test-portal-source-format/3353/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-22.liferay.com//userContent/jobs/test-portal-source-format/builds/3353/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 2 minutes 27 seconds 386 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/2e0862286535b794943a0b519da71f7bff312691">2e0862286535b794943a0b519da71f7bff312691</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/matthewchan1/liferay-portal/tree/LPS-100257">LPS-100257</a><br/>Branch GIT ID: <a href="https://github.com/matthewchan1/liferay-portal/commit/f6c077a8b90c9adc0b574a66bd6c2b3cac63f3b7">f6c077a8b90c9adc0b574a66bd6c2b3cac63f3b7</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-22.liferay.com/job/test-portal-source-format/3353/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-22.liferay.com/userContent/jobs/test-portal-source-format/builds/3353/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 2 minutes 55 seconds 164 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/2e0862286535b794943a0b519da71f7bff312691">2e0862286535b794943a0b519da71f7bff312691</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/matthewchan1/liferay-portal/tree/LPS-100257">LPS-100257</a><br/>Branch GIT ID: <a href="https://github.com/matthewchan1/liferay-portal/commit/f6c077a8b90c9adc0b574a66bd6c2b3cac63f3b7">f6c077a8b90c9adc0b574a66bd6c2b3cac63f3b7</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-4-1.liferay.com/job/test-portal-source-format/1329/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-4-1.liferay.com/userContent/jobs/test-portal-source-format/builds/1329/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest(master)/4938/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-22.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4938/jenkins-report.html">here</a>.</b></p>

brianikim commented:
ci:stop

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 0 out of 1 jobs passed in 8 minutes 31 seconds 896 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/0d2b46300a6d0ec4ac2965c18fb87056b1272986">0d2b46300a6d0ec4ac2965c18fb87056b1272986</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/d80fd3ca7354bc36340d6768efba4904f2b1c8e2">d80fd3ca7354bc36340d6768efba4904f2b1c8e2</a></p>0 out of 1 jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest(master)/4938/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-22.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4938/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest(master)/4938/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><pre><code>Build was aborted</code></pre></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></details></html>

mbowerman committed:
LPS-170141 Revert "LPS-156243 We need to still return the UserGroup even when the key was"

This reverts commit 8876f380f42f95185b2cc0edd3fd2c458ea93ea9.

mbowerman committed:
LPS-170141 Revert "LPS-142044 SF"

This reverts commit da4dbc86bb610ee4357fd0c097b0284e01c3a2fd.

mbowerman committed:
LPS-170141 Revert "LPS-142044 Avoid passing a user object simply to make a log statement (which also might not be truthful)"

This reverts commit c59f02c1e448c88b7ddb6374b73aa57e79102dee.

mbowerman committed:
LPS-170141 Revert "LPS-142044 These can be outside the while loop - they're not going to"

This reverts commit de75fdd539734ec996ff04c8a6641636e3df9abf.

mbowerman committed:
LPS-170141 Revert "LPS-142044 More accurate name"

This reverts commit 90999150576905058b89829c04cbeafa39a98095.

mbowerman committed:
LPS-170141 Revert "LPS-142044 Also add user groups that were added by LDAP import from a"

This reverts commit f5bd96f38ef8a900545fa2960205fef90938842d.

mbowerman committed:
LPS-170141 Revert "LPS-142044 Import all groups belonging to the server during this process and track two separate Sets: one continaing all the groups belonging to the server, and another containing all the groups the user belongs to within this server."

This reverts commit 23b4d2119f4f78726fb5f9d4905342a722c63303.

mbowerman committed:
LPS-170141 Revert "LPS-142044 Refactor importGroup method to return the UserGroup and have the caller be responsible for adding it to the Set that it wants. No logic change yet."

This reverts commit 027e316605b9f053ef52a39fe1aec447f94e30e0.

Commit:
LPS-171009 Add objectDefinitionERC property and Create GET and POST ObjectValidationRules by objectDefinitionERC

Commit:
LPS-171009 buildRest

Commit:
LPS-171009 Implement endpoint methods and adjust toObjectValidationRule

Commit:
LPS-171009 Implement integration tests

AlmirFe committed:
LPS-171073 Add another way to get layout name in case name is null

JanisZhang committed:
LPS-171138 Remove Stream usages in Changeset

JanisZhang committed:
LPS-171138 Apply usages

sandrodw3 committed:
LPS-171392 Order

CrescenzoRegaLF committed:
COMMERCE-10506 Autogenerated code (blade gw buildService)

CrescenzoRegaLF committed:
COMMERCE-10506 Semantic versioning (gw baseline)

CrescenzoRegaLF committed:
COMMERCE-10506 - commerce-product-service - Adapt the code to the new logic

CrescenzoRegaLF committed:
COMMERCE-10506 - commerce-product-item-selector-web - Adapt the code to the new logic

CrescenzoRegaLF committed:
COMMERCE-10506 - commerce-channel-web - Add CommerceChannelControlPanelEntry

murilostodolni committed:
LPS-169369 Add notificationTemplateERC and type when export ObjectDefinition

murilostodolni committed:
LPS-169369 Create method to add NotificationTemplate with ERC

sandrodw3 committed:
LPS-171392 Add support for getting active item id from URL

sandrodw3 committed:
LPS-171392 Add new method to get a collection's edit configuration url

sandrodw3 committed:
LPS-171392 Filter button will navigate to new configuration page

murilostodolni committed:
LPS-169369 Build service

murilostodolni committed:
LPS-169369 Implement method to add NotificationTemplate by ERC

murilostodolni committed:
LPS-169369 Validate Notification ObjectAction

claraizquierdo committed:
LPS-167150 Use new Picker component for experience selector

guilhermedcamacho committed:
LPS-169369 Inline and Sort

guilhermedcamacho committed:
LPS-169369 Sort

guilhermedcamacho committed:
LPS-169369 regen

claraizquierdo committed:
LPS-167150 Style improvements in Experience Picker

sandrodw3 committed:
LPS-171392 Add CollectionConfiguration component

sandrodw3 committed:
LPS-171392 Move useSelector to AdvancedSelectField so SelectField component doesn't depend on state

sandrodw3 committed:
LPS-171392 Move useSelector outside so we can use FieldSet without depending on state

claraizquierdo committed:
LPS-167150 Disable experience picker if simulation panel is open

Commit:
LPS-170888 Fix the fix wrong logic

Commit:
LPS-170888 Convert method reference to ServiceTrackerMap

claraizquierdo committed:
LPS-167150 Remove unnecessary js code

murilostodolni committed:
LPS-169369 Add test integration

claraizquierdo committed:
LPS-167150 Accessibility improvements in ExperiencePicker

murilostodolni committed:
LPS-169369 Add userId and userName in NotificationRecipient and NotificationRecipientSetting

tinatian committed:
LPS-170888 SF, simplify

JanisZhang committed:
LPS-171138 Remove the deprecated methods in ChangesetManager and its impl

JanisZhang committed:
LPS-171138 Remove Optional usages in PortletExportControllerImpl

JanisZhang committed:
LPS-171138 Remove Optional usages in _getHttpPrincipal method in EventRemotePropagatorExportImportLifecycleListener

JanisZhang committed:
LPS-171138 Remove Optional usages in EventRemotePropagatorExportImportLifecycleListener

JanisZhang committed:
LPS-171138 SF

JLucasM committed:
LRAC-12673 Create unit tests for Wizard Page and their Steps

JLucasM committed:
LRAC-12673 TD Generated files

sandrodw3 committed:
LPS-171392 defaultValue could be an array

sandrodw3 committed:
LPS-171392 Initialize value

sandrodw3 committed:
LPS-171392 Use default values, so components do not break when using them outside of the context

JanisZhang committed:
LPS-172727 Remove Optional and Stream usages in OpenGraphTopHeadDynamicInclude

JanisZhang committed:
LPS-172727 Remove Optional usages in OpenGraphImageProvider

cgoncas committed:
LPS-167150 Better if we create a getData method inside SegmentsExperienceSelectorDisplayContext with the experiences and  selectedExperience values.

cgoncas committed:
LPS-167150 Update tests.

Stephy6 committed:
LPS-167158 Add title to ExperiencePicker to display pop-up label

Stephy6 committed:
LPS-167158 buildLang for 'Experience Selector' string

Commit:
LPS-170141 Revert "LPS-156599 SF"

This reverts commit f330617e6a0b1bd9cb3569f86ff4b81d062eb495.

Commit:
LPS-170141 Revert "LPS-156599 Handle null and multiple values more carefully"

This reverts commit 3f83ce9b7bd656ef1ba8b2d4f06f73198d207f74.

Commit:
LPS-170141 Revert "LPS-156599 Set values of userAttribute and fullUserDN to lower case so the condition can be case insensitive"

This reverts commit 9e130155bb426a286e000af35398bd08115a68db.

Commit:
LPS-170141 Revert "LPS-156243 We cannot rely on the member attribute of the group object,"

This reverts commit 9c85fe5006396d8201972a08ba47c0dc2c484d62.

tinatian committed:
LPS-171138 Remove deprecated methods

tinatian committed:
LPS-171138 Remove unused method

tinatian committed:
LPS-171138 SF, simplify

tinatian committed:
LPS-171138 Shortcut earlier to improve performance

carolmariaabb committed:
LPS-172282 Change the Field Name and Option Value based on the Field and Option Reference in Web Content Structures

sandrodw3 committed:
LPS-171392 Don't calculate synthetic items when we are not in page editor context

sandrodw3 committed:
LPS-171392 Initialize config so we have namespace available in ItemSelector component

sandrodw3 committed:
LPS-171392 Add support for dependency

Mikellorza committed:
LPS-171998 Support SUB binary expression operator in Odata API

claraizquierdo committed:
LPS-167150 Rename properties

sandrodw3 committed:
LPS-171392 Add support for description

sandrodw3 committed:
LPS-171392 Change copy

sandrodw3 committed:
LPS-171392 Support inline selects

sandrodw3 committed:
LPS-171392 Use AND as it's an operator

Mikellorza committed:
LPS-171998 Support DURATION type in Odata API

carolmariaabb committed:
LPS-172282 Add unit test

Commit:
LPS-172486 change messageKey

Commit:
LPS-172486 create new language key

Commit:
LPS-172486 buildLang

Commit:
LPS-167280 Add liferay npm scripts for site-initializer-liferay-lean

thienquach87 committed:
LPS-172528 Root Folder configuration in the Document & Media widget is lost when using Category Filter

Mikellorza committed:
LPS-171998 Implement Sub arithmetic Operation

Mikellorza committed:
LPS-171998 Add test

Commit:
LPS-172486 Update message in poshi test

interaminense committed:
LRAC-12673 Implement click to finish step and update pageView mode

JanisZhang committed:
LPS-172727 No need to use if-statement while add value to set

JanisZhang committed:
LPS-172727 Remove Optional usages in ContentPageEditorDisplayContext

JanisZhang committed:
LPS-172727 Remove Optional usages in LayoutReportsProductNavigationControlMenuEntry

JanisZhang committed:
LPS-172727 Remove Optional usages in LayoutReportsDataMVCResourceCommand

drewbrokke committed:
LPS-146722 roles-test: adds additional assertions to ensure the behavior of the method is not broken by optimizations

Brunno0liveira committed:
COMMERCE-10560 Add AssertMiniCartItemAllowedQuantities Macro

Brunno0liveira committed:
COMMERCE-10560 Add MINI_CART_ITEM_QUANTITY_SELECTOR_SELECT_FIELD Path

Brunno0liveira committed:
COMMERCE-10560 Allowed quantities quantity can be added to cart/Allowed quantities quantity cannot be added to cart And Change test to use API in all testcase

pietrobocale committed:
COMMERCE-10602 Add new macro defineProductPermissionsViaJSONAPI

pietrobocale committed:
COMMERCE-10602 Add new test CanUserWithChannelViewPermissionSeeChannelEntries

jeffwu0724 committed:
LPS-172718 remove usage of stream in GetItemDetailsMVCResourceCommand

jeffwu0724 committed:
LPS-172718 remove usage of stream in DisplayPageTypeSiteNavigationMenuTypeDisplayContext

jeffwu0724 committed:
LPS-172718 remove usage of Optional in DisplayPageTypeContext

jeffwu0724 committed:
LPS-172718 apply usage

JanisZhang committed:
LPS-172727 Simplify, getSegmentsExperienceId should never return null

JanisZhang committed:
LPS-172727 Remove Stream usages in ContentPageEditorDisplayContext

JanisZhang committed:
LPS-172727 Remove Optional usages in WebImage

JanisZhang committed:
LPS-172727 Apply usages

JanisZhang committed:
LPS-171138 Sem Ver

sandrodw3 committed:
LPS-171392 buildLang

CarlosBrichete committed:
LPS-167150 Fix macro for experience selector

lfbesada committed:
LPS-172778 When rendering a form do not display it if the user hasn't the ADD permission, no matter the layout mode

lfbesada committed:
LPS-172778 SF unused

lfbesada committed:
LPS-172778 SF

murilostodolni committed:
LPS-169369 baseline

dantewang committed:
LPS-172718 SF, unnecessary conversion

austinchiang committed:
LPS-166603 Don't delete liferay home when rebuilding the DB

JanisZhang committed:
LPS-172727 Sem Ver

lfbesada committed:
LPS-172409 Add Page Sets page's type to the search

lfbesada committed:
LPS-172409 SF extract and reuse

lfbesada committed:
LPS-172409 Auto

lfbesada committed:
LPS-169674 Change the usage to relate it to the CollectionStyledLayoutStructureItem instead of the LayoutPageTemplateStructure

lfbesada committed:
LPS-169674 Add new finder by containerType and plid

lfbesada committed:
LPS-169674 Add new methods to get and delete by containerType and plid

lfbesada committed:
LPS-169674 BuildService

lfbesada committed:
LPS-169674 Use the new deleteAssetListEntryUsages method to remove the previous AssetListEntryUsages

lfbesada committed:
LPS-169674 Improve the performance, use fields to cache the classNameIds

lfbesada committed:
LPS-169674 Remove the redundant checking.

Now we are relating the usage to the itemId, which is unique in the page (the fragmentEntryLinkId has been always unique), and we previously deleted all this type's usage of this page. So this condition will never occur, unless concurrency in the same layout's edition, which is not supported.

lfbesada committed:
LPS-169674 Add the containerKey to the unique key

lfbesada committed:
LPS-169674 Simplify

lfbesada committed:
LPS-169674 SF extract to variable and reuse

lfbesada committed:
LPS-169674 Take into account that the CollectionStyledLayoutStructureItem could be marked for deletion

lfbesada committed:
LPS-169674 SF extract to static fields and reuse

lfbesada committed:
LPS-169674 Adds an upgrade process

lfbesada committed:
LPS-169674 SF the redirect is the same for all the usages, prevent from calculating it one and again

lfbesada committed:
LPS-169674 Register the upgrade step

lfbesada committed:
LPS-169674 Baseline

Commit:
LPS-172614 Fix router on LXC env

Commit:
LPS-167280 Add liferay npm scripts for frontend-css-cadmin-web

Commit:
LPS-167280 Add liferay npm scripts for frontend-css-common

Commit:
LPS-167280 Fix the issue which found with SF

smottal committed:
COMMERCE-10579 Return ERC attribute on products APIs (yaml)

smottal committed:
COMMERCE-10579 Return ERC attribute on products APIs

smottal committed:
COMMERCE-10579 Semantic versioning

smottal committed:
COMMERCE-10579 Autogenerated Code (gw buildRest)

sandrodw3 committed:
LPS-171392 This is no needed, it was introduced for LPS-99490 and it's not happening anymore

AliciaGarciaGarcia committed:
LPS-172863 on the edition show the description field

AliciaGarciaGarcia committed:
LPS-172863 show the description panel on the view of the article

AliciaGarciaGarcia committed:
LPS-172863 view the article description only if is enabled and is not empty

Commit:
LPS-172811 object-web: Add companyId property so when has DB Partition AssetRenderFactory could be filtered by companyId

Commit:
LPS-172811 portal-search: Simplify

Commit:
LPS-172811 portal-search-web: Prevent NPE

Commit:
LPS-172811 portal-kernel: Filter AssetRenderFactories with new property companyId to inlcude use case when DB Partition

Commit:
LPS-172811 SF

AliciaGarciaGarcia committed:
LPS-172863 on searching the value of the attribute is lost so we must recreate it

kyle-miho committed:
LPS-168452 Temporarily ignore failing test

jaydawu committed:
LRQA-74780 Add CanViewFieldsAtDetailsTab test

lfbesada committed:
LPS-172215 Add an util to get the PortletRegistry reference

Mikellorza committed:
LPS-171998 Return MethodType.NOW type instead of String when visiting Now() odata method.

naraandrade committed:
LPS-173011 Enable changing Condition node's scriptLanguage through diagram view

lfbesada committed:
LPS-122215 Hide the content's usages within the items contained by those Form Container that the user hasn't VIEW permissions

lfbesada committed:
LPS-122215 Hide the portlets usages that are only within items contained by those Form Containers that the user hasn't VIEW permissions

If the same portlet instance is in, at least, one no-restricted item, its usage should be visible

lfbesada committed:
LPS-122215 Hide the restrictions under the feature flag

lfbesada committed:
LPS-122215 SF extract to static fields

lfbesada committed:
LPS-122215 Pass the restrictedItemIds and use it to hide the collections usages within the items contained by those Form Container that the user hasn't VIEW permissions

riccardo-ferrari committed:
LRAC-12785 analytics-settings-rest-api Update Account fields

Commit:
LPS-173196 Add Workaround due to LPS-173196 for skiping virtual instance on Raylife D2C tests

Commit:
LPS-173196 Add Raylife D2C mobile tests to quarentine

Commit:
LPS-173196 SF

ericyanLr committed:
LPS-167054 Allow scope to be configurable for the Related Assets portlet

Expected behavior:
- 7.0 and below:
  - Default Behavior: Return related assets from every scope
  - Scope: Not configurable
- 7.1 and above:
  - Default Behavior: Only return related assets from current site
  - Scope: Configurable

History:
- 7.0 and below:
  - Default Behavior: Return related assets from every scope
  - Scope: Not configurable
- 7.1 and above (before LPS-151347 - 9fc2c8ef)
  - Default Behavior: Only return related assets from current site
  - Scope: Configurable
- (after LPS-151347 - 9fc2c8ef)
  - Default Behavior: Only return related assets from current site
  - Scope: Not configurable

JLucasM committed:
LRAC-12673 Fix the failing on tests

JLucasM committed:
LRAC-12673 Create a property data-testid for use on tests

JLucasM committed:
LRAC-12673 TD Generated files

Commit:
LPS-172811 portal-kernel: Fix colateral effects from the new filter to _filterAssetRendererFactories

drewbrokke committed:
LPS-146722 roles-test: assert that group roles are not returned when no groups are passed in

drewbrokke committed:
LPS-146722 portal-impl: if no groupIds are passed in, skip the queries on Group_Roles

drewbrokke committed:
LPS-146722 portal-impl: updates the javadoc to represent the new behavior

Commit:
LPS-169774 Convert method reference to ServiceTrackerList(The inner class SortableHitsProcessor is used to provide a comparison function for sorting, and we can replace it with method ServiceTrackerListFactory.open(BundleContext bundleContext, Class<T> clazz,Comparator<ServiceReference<T>> comparator))

austinchiang committed:
LPS-170173 Split into smaller tests

dantewang committed:
LPS-172727 SF, Language.getAvailableLocales(groupId) never returns null

dantewang committed:
LPS-172727 SF, simplify

dantewang committed:
LPS-172727 SF, make it clear for vararg

dantewang committed:
LPS-172727 SF, proper conversion

austinchiang committed:
LPS-170173 Sort

austinchiang committed:
LPS-170173 Use JSON to enable staging

austinchiang committed:
LPS-170173 Add document through JSON

austinchiang committed:
LPS-170173 Remove bloat

austinchiang committed:
LPS-170173 Add document via JSON

austinchiang committed:
LPS-170173 Rename tests

Mikellorza committed:
LPS-172394 Text SF

ambrinchaudhary committed:
LPS-173256 Add lazy loading for panels

Commit:
LPS-173263 Small description field positioning adjustment

lfbesada committed:
LPS-172774 Prevents from displaying an empty configuration view when the InfoCollectionProvider is configurable but its configuration InfoForm is empty.

That is a real case that occurs with the OOTB collections of any site-scoped ObjectEntry if the Definition doesn't allow any classification (categories or tags)

AliciaGarciaGarcia committed:
LPS-172863 on section the value of the attribute is lost too

ruben-pulido committed:
LPS-173244 Remove unused reference to AssetEntryLocalService

baharturk committed:
COMMERCE-10599 Fix failing test

lfbesada committed:
LPS-122215 SF sort

Commit:
LPS-173265 Add WorkflowMessageBoards tests to quarantine

lfbesada committed:
LPS-122215 SF rename

JLucasM committed:
LRAC-12673 Create '__tests__' folder on wizard

lfbesada committed:
LPS-122215 Fix the condition, it should be an AND

drewbrokke committed:
LPS-146722 portal-impl: makes the behavior consistent across overloaded methods

drewbrokke committed:
LPS-146722 liferay-portal: autogenerated: build-service-portal

thiagobuarque-liferay committed:
LRAC-12602 addapt comparator to XML fields with locale

Commit:
LPS-173196 fixing

sandrodw3 committed:
LPS-171392 Adapt tests

Commit:
LPS-173196 SF

JLucasM committed:
LRAC-12673 Delete unnecessary tests and their ts files

kenjiheigel committed:
LRCI-3320 Add qa-websites validation to poshi release tester

CalumR23 committed:
POSHI-463 Allow for non quoted value in ElementAttributes

kenjiheigel committed:
POSHI-463 Use com.liferay.poshi.core:1.0.91

JoyceWang08211 committed:
LRQA-79110 Update case

ling-alan-huang committed:
LPS-170079 SF, inline

sergiojimcos committed:
LPS-172697 Rename tests related getting object relationship ERC field name

victorg1991 committed:
LPS-172873 Take into account the selPpid parameter too

lipusz committed:
LPS-173323 portal-search-solr8: Add initial .releng files

https://issues.liferay.com/browse/LPS-173323

Signed-off-by: Tibor Lipusz <tibor.lipusz@liferay.com>

sandrodw3 committed:
LPS-172914 Remove top margin

Commit:
LPS-173330 etl-mulesoft - update dependency versions to compatible ones

riccardo-ferrari committed:
LRAC-12785 analytics-batch-export-import-impl Disable permission checking on export

ambrinchaudhary committed:
LPS-173256 UseMemo to Cache normalized actions recursively

xbrianlee committed:
RELEASE-5403 Liferay Connector to Solr 8 5.0.0 master (releng)

carolmariaabb committed:
LPS-173373 Add new method to ObjectEntryManager

carolmariaabb committed:
LPS-173373 baseline

carolmariaabb committed:
LPS-173373 Move logic from ObjectEntryResource to ObjectEntryManager

carolmariaabb committed:
LPS-173373 If the user has permission to trigger the Object Action, the user doesn't need permission with the Object Entry that the Action interacts with

brianchandotcom committed:
LRAC-12602 SF

brianchandotcom committed:
LPS-172727 as used

brianchandotcom committed:
LRCI-3320 prep next

brianchandotcom committed:
LPS-172528 SF

brianchandotcom committed:
POSHI-463 SF

brianchandotcom committed:
POSHI-463 prep next

brianchandotcom committed:
LPS-172811 Rename

brianchandotcom committed:
LPS-172811 auto SF

brianchandotcom committed:
LPS-172863 SF

liferay-continuous-integration committed:
subrepo:ignore Update 'modules/etl/mulesoft/.gitrepo'.

brianchandotcom committed:
LPS-171998 Wordsmith

brianchandotcom committed:
LPS-171998 Fix space

brianchandotcom committed:
LPS-171998 Wordsmith

brianchandotcom committed:
LPS-171998 Simplify

brianchandotcom committed:
LPS-171998 SF

brianchandotcom committed:
Revert "LPS-171392 Use AND as it's an operator"

This reverts commit 591192dd041acd541058ed973b429c9519db31de.

brianchandotcom committed:
Revert "LPS-171392 buildLang"

This reverts commit 4ebff2d4e4084ed5dd02e28a3fc3a1917d2687f2.

brianchandotcom committed:
LRAC-12785 Sort

brianchandotcom committed:
LPS-171138 Buy space

brianchandotcom committed:
LPS-166603 Sort #sfme

brianchandotcom committed:
LPS-154744 Sort

brianchandotcom committed:
LPS-169369 Consistency. I noticed this pattern that you added which I slightly modified in ObjectDefinitionLocalServiceTest, but you did not apply those modifications to NotificationTemplateLocalServiceTest. Please review my changes and apply them to all future code. See 5a1836da8eaf8eab7cc37711aac9afc5d1dbeed5/

brianchandotcom committed:
COMMERCE-10579 SF

brianchandotcom committed:
LPS-154677 apply here

Commit:
COMMERCE-10243 Add default value for Specification Key when Patching

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

victorcdp commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/e9e56ed1a9b52ca6675823edffe0c66a2318a94b">e9e56ed1a9b52ca6675823edffe0c66a2318a94b</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorcdp/liferay-portal/tree/COMMERCE-10243">COMMERCE-10243</a><br/>Branch GIT ID: <a href="https://github.com/victorcdp/liferay-portal/commit/47096f7e9d4ab4ad0382b873f3a7cfd8ab49f103">47096f7e9d4ab4ad0382b873f3a7cfd8ab49f103</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-2.liferay.com/job/test-portal-source-format/1300/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-2.liferay.com/userContent/jobs/test-portal-source-format/builds/1300/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-2.liferay.com/job/test-portal-source-format/1300/">test-portal-source-format#1300</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-2.liferay.com/userContent/jobs/test-portal-source-format/builds/1300/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/46">brianikim#46</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2170494285">[master] ci:test:sf - victorcdp &gt; brianikim - PR#46 - 2023-01-23[08:19:48]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-2.liferay.com/job/publish-testray-report/3624/">publish-testray-report#3624</a><br/></div></div>
